File: app/src/main/java/ru/vincetti/vimoney/data/models/TransactionStatDayModel.java
package ru.vincetti.vimoney.data.models;

public class TransactionStatDayModel {

    private int day;
    private int sum;

    public void setDay(int day) {
        this.day = day;
    }

    public void setSum(int sum) {
        this.sum = sum;
    }

    public int getDay() {
        return day;
    }

    public int getSum() {
        return sum;
    }
}

File: database/src/main/java/ru/vincetti/modules/database/sqlite/models/TransactionStatDayModel.kt
package ru.vincetti.modules.database.sqlite.models

import ru.vincetti.modules.core.models.TransactionStatDay

data class TransactionStatDayModel(
    val day: String,
    val sum: Int
) {

    fun toTransactionStatDay(): TransactionStatDay {
        return TransactionStatDay(day, sum)
    }
}
